content/tutorials/get-started/monitor-incoming-payments-with-websocket.md

```diff
@@ -0,0 +1,476 @@
+do_subscribe()
+
+const log_tx = function(tx) {
+  console.log(tx.transaction.TransactionType + " transaction sent by " +
+              tx.transaction.Account +
+              "\n  Result: " + tx.meta.TransactionResult +
+              " in ledger " + tx.ledger_index +
+              "\n  Validated? " + tx.validated)
+}
+WS_HANDLERS["transaction"] = log_tx
+```
+
+For the following example, try opening the [Transaction Sender](tx-sender.html) in a different window or even on a different device and sending transactions to the address you subscribed to:
+
+{{ start_step("Subscribe") }}
+<label for="subscribe_address">Test Net Address:</label>
+<input type="text" class="form-control" id="subscribe_address" value="rUCzEr6jrEyMpjhs4wSdQdz4g8Y382NxfM">
+<button id="tx_subscribe" class="btn btn-primary" disabled="disabled">Subscribe</button>
+<h5>Transactions</h5>
+<div class="ws-console" id="monitor-console-subscribe"><span class="placeholder">(Log is empty)</span></div>
+{{ end_step() }}
+
+<script type="application/javascript">
+async function do_subscribe() {
+  const sub_address = $("#subscribe_address").val()
+  const sub_response = await api_request({
+    command:"subscribe",
+    accounts: [sub_address]
+  })
+  if (sub_response.status === "success") {
+    writeToConsole("#monitor-console-subscribe", "Successfully subscribed!")
+  } else {
+    writeToConsole("#monitor-console-subscribe",
+                   "Error subscribing: " + JSON.stringify(sub_response))
+  }
+}
+$("#tx_subscribe").click((event) => {
+  do_subscribe()
+  complete_step("Subscribe")
+  $("#tx_read").prop("disabled", false)
+})
+
+const log_tx = function(tx) {
+  writeToConsole("#monitor-console-subscribe",
+                  tx.transaction.TransactionType + " transaction sent by " +
+                  tx.transaction.Account +
+                  "<br/>&nbsp;&nbsp;Result: " + tx.meta.TransactionResult +
+                  " in ledger " + tx.ledger_index +
+                  "<br/>&nbsp;&nbsp;Validated? " + tx.validated)
+}
+WS_HANDLERS["transaction"] = log_tx
+</script>
+
+
+## {{n.next()}}. Read Incoming Payments
+
+When you subscribe to an account, you get messages for _all transactions to or from the account_, as well as _transactions that affect the account indirectly_, such as trading its [issued currencies](issued-currencies.html). If your goal is to recognize when the account has received incoming payments, you must filter the transactions stream and process the payments based on the amount they actually delivered. Look for the following information:
+
+- The **`validated` field** indicates that the transaction's outcome is [final](finality-of-results.html). This should always be the case when you subscribe to `accounts`, but if you _also_ subscribe to `accounts_proposed` or the `transactions_proposed` stream then the server sends similar messages on the same connection for unconfirmed transactions. As a precaution, it's best to always check the `validated` field.
```

Comment:
A nit -- but when I look at the generated HTML, it looks like only "field" is bolded. The monospace field name before "field" is bolded so lightly, you can't tell is it bold. Users may wonder why "field" is bolded.

Reply:
I'm not seeing it like that on my system, but that's not all that surprising, as the font and amount of bolding that gets effectively used may vary depending on your system and our CSS. It might be possible to adjust the CSS to make bolded code-font more prominent on more systems, but I think that's out of scope for this PR.